decidim/decidim
# Debates with no start and end time

#### :tophat: What? Why?
This feature allows for admins to create debates with no start and end times, and display "Open debate" on the public debate cards.

#### :pushpin: Related Issues
- Related to [MetaDecidim Proposal \#15688](https://meta.decidim.org/processes/roadmap/f/122/proposals/15688)

#### Testing
*Create an open-dated debate*
- Visit the debates component for any participatory space in the admin
- Create a new debate
- Choose "Open (No start or end time)"
- Submit the form by clicking on "Create"
- Check that the debate has no start or end time
- Visit the public section for debates
- Check that debate card has "Open debate" in the date section

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots

#### Open Debates - Admin

![Open Debates - Admin](https://user-images.githubusercontent.com/8806781/100746217-b490a000-33e0-11eb-9f6c-4058064c36e9.gif)


#### Open Debates - Public

![Open Debates - Public](https://user-images.githubusercontent.com/8806781/100729454-134a1f80-33c9-11eb-9842-7a8b0c476037.png)

:hearts: Thank you!
